packages/common/src/types.ts

```diff
@@ -40,8 +40,11 @@ export interface Chain {
   }
 }
 
+export type StoragePair = [key: string, value: string]
```

Comment:
maybe we use `PrefixedHexString` for increased clarity? can also use in AccountState

Reply:
+1 for `PrefixedHexString` (this is in Util `types.ts`)

It is generally good if we try to add new exports as cautiously as possible, since every new export becomes part of the API and this becomes difficult to change later on (since many changes then become "breaking" changes, e.g. if we would want to rename `StoragePair` and someone uses it this is then breaking people's code.

Just as some context, concretely here: do we need to expose `StoragePair` here? I guess this is rather taken implicitly in in `AccountState` and external libraries/users won't need this for typing? 🤔